components/dashboard/src/login/SSOLoginForm.tsx

```diff
@@ -66,23 +52,29 @@ export const SSOLoginForm: FC<Props> = ({ onSuccess }) => {
     );
 
     // Don't render anything if not enabled
-    if (!showSSO) {
+    if (!oidcServiceEnabled) {
         return null;
     }
 
     return (
         <form onSubmit={openLoginWithSSO}>
             <div className="mt-10 space-y-2">
-                <TextInputField
-                    label="Organization Slug"
-                    placeholder="my-team"
-                    value={orgSlug}
-                    onChange={setOrgSlug}
-                    error={slugError.message}
-                    onBlur={slugError.onBlur}
-                />
-                <Button className="w-full" type="secondary" disabled={!orgSlug.trim() || !slugError.isValid}>
-                    Continue with SSO
+                {!singleOrgMode && (
+                    <TextInputField
+                        label="Organization Slug"
+                        placeholder="my-company"
+                        value={orgSlug}
+                        onChange={setOrgSlug}
+                        error={slugError.message}
+                        onBlur={slugError.onBlur}
+                    />
+                )}
+                <Button
+                    className="w-full"
+                    type="secondary"
+                    disabled={!singleOrgMode && (!orgSlug.trim() || !slugError.isValid)}
+                >
+                    Continue {singleOrgMode ? "" : "with SSO"}
```

Comment:
In this case there are no other ways to continue so I don't think the "with SSO" part is helpful. In single org-mode (dedicated), we don't need to talk about the org a lot because it is the only context that exists. With only one button I believe just "Continue" is best.

Reply:
Fair. FWIW when testing I refrained to hot "Continue" directly because it was not that clear what's happening. But that might have been just the first time as well. :lotus_position: